src/pages/ProblemSubmitView.vue

```diff
@@ -40,11 +45,31 @@ const router = useRouter();
 const route = useRoute();
 
 const problemNo = route.params.no;
-const languageList: string[] = ["C"];
+const receivedProblems = await getAsync<any>("/problem/" + problemNo);
+const problem = ref(receivedProblems.data as Problem);
+const languageList: string[] = ["C99", "C++17"];
+const selectedLanguage = ref(languageList[0]);
 
 const form = ref();
+const textarea = ref();
 
-const selectedLanguage = ref(languageList[0]);
+const useTab = (event: any) => {
```

Comment:
이렇게 하니까 커서가 정상적인 위치로 가네요
```suggestion
const useTab = (event: KeyboardEvent) => {
  const start: number = textarea.value.selectionStart;
  const end: number = textarea.value.selectionEnd;

  textareaCode.value =
    textareaCode.value.substring(0, start) +
    "\t" +
    textareaCode.value.substring(end, textareaCode.value.length);

  setTimeout(() => {
    (textarea.value as HTMLTextAreaElement).setSelectionRange(
      start + 1,
      start + 1
    );
  }, 0);
};
```